cinder/volume/drivers/lightos.py

```diff
@@ -1364,8 +1364,11 @@ def _create_snapshot(self, project_name, snapshot_name, src_volume_name):
                     src_volume_name=src_volume_name,
                 )
 
-                if status_code_create == httpstatus.INTERNAL_SERVER_ERROR:
-                    pass
+                if (status_code_create == httpstatus.BAD_REQUEST or
+                       status_code_create == httpstatus.INTERNAL_SERVER_ERROR):
+                    LOG.info('Snapshot %s creation status %s', snapshot_name,
```

Comment:
This will spam the logs like there's no tomorrow. Please change to `LOG.debug()` and consider adding the pertinent info that will enable you debug it: project_name, snapshot_name, src_volume_name.

Reply:
p.s had a look at our K8 support seems that there we have dedicated code that converts the bad request (fail preondition) to unavialable

Reply:
@ronen-lb I think it's  better to keep your PR and merge into duros, i will add this new code SERVICE_UNAVAILABLE to cinder driver.